docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -169,15 +168,16 @@ The reasons that DNS service is per-network instead of per-space are:
 
 #### Type definition
 
-Addition of service spec is spec.network field. By default, it is the default network of the tenant. 
+Service objects associate to network via label arktos.futurewei.com/network. By default, network is the default network of the tenant.
```

Comment:
In mizar, service acts like a load balancer, we call it scaled endpoint.
Will we be provisioning a default number of backends for this service/scaled ep?
Will we allow the user to specify how many backends they desire for their service/scaled ep?

Reply:
how many backends are determined by the number of endpoints in arktos(k8s). That number could be dynamic. please make sure mizar is able to handle that case.